libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerOptions.cs

```diff
@@ -113,12 +114,35 @@ public QnAMakerOptions()
         public string RankerType { get; set; }
 
         /// <summary>
-        /// Gets or sets <see cref="StrictFilters"/> join operator.
+        /// Gets or sets strictFiltersJoinOperator - join operator on metadata strict filters.
         /// </summary>
-        /// <value>
-        /// A value used for join operation of StrictFilters <see cref="StrictFilters"/>.
-        /// </value>
+        /// <value>StrictFiltersJoinOperator - join operator on metadata strict filters.</value>
         [JsonProperty("strictFiltersJoinOperator")]
         public JoinOperator StrictFiltersJoinOperator { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether enable Precise Answer.
+        /// </summary>
+        /// <value>
+        /// To enable Precise Answer or not.
+        /// </value>
+        [JsonProperty("enablePreciseAnswer")]
+        public bool EnablePreciseAnswer { get; set; }
+
+        /// <summary>
+        /// Gets or sets the metadata and sources used to filter QnA Maker results.
+        /// </summary>
+        /// <value>
+        /// An object with metadata, source filters and corresponding operators.
+        /// </value>
+        [JsonProperty("filters")]
+        public Filters Filters { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether Flag to enable Query over Unstructured Sources (Optional).
+        /// </summary>
+        /// <value> Flag to enable Query over Unstructured Sources.</value>
```

Comment:
>Flag to enable Query over Unstructured Sources. [](http://example.com/codeflow?start=20&length=47)

Indicate what the possible values are, and what they mean. Also, include the default value here.